src/main.tsx

```diff
@@ -4,42 +4,50 @@ import './index.css'
 import { Provider } from 'react-redux'
 import { setupStore } from './redux/store.tsx'
 import { createBrowserRouter, RouterProvider } from 'react-router-dom'
-import { Outlet } from 'react-router-dom'
+import WorldInformation from './components/WorldInformation.tsx'
+
+
+//import { Outlet } from 'react-router-dom'
 import Navbar from './components/Navbar'
 import Consult from './components/Consult.tsx'
 const NavbarWrapper = () => {
   return (
   <div>
       <Navbar/>
-      <Outlet/>
+      <WorldInformation/>
```

Comment:
# Really nice this component!!! 
I recomment place `<Outlet />` here and let it make :sparkles: